src/main/java/org/irods/operations/UserGroupOperations.java

```diff
@@ -114,6 +211,12 @@ public Response users(String token) {
         return new Response(response.statusCode(), response.body());
     }
 
+    /**
+     * Lists all groups in the zone. This operation requires rodsadmin level privileges.
```

Comment:
does this require admin?  hmmm..

Reply:
That's what the API documentation was saying :/

Reply:
attn @korydraughn 

not an emergency - just interesting.   as this information can be retrieved via query, regardless.

Reply:
It may not require rodsadmin privileges, but I'd have to confirm that.

Can always let it ride and put it in an issue for investigation.

Reply:
Created #98 for this.